regparser/commands/versions.py

```diff
@@ -68,6 +72,7 @@ def write_if_needed(cfr_title, cfr_part, version_ids, xmls, delays):
     version_dir = entry.Version(cfr_title, cfr_part)
     deps = generate_dependencies(version_dir, version_ids, delays)
     for version_id in version_ids:
+        logger.debug("Writing version %r", version_id)
```

Comment:
As noted earlier, we already log when writing to disk. More important here, though -- we might not always be writing the version at this point -- we'd only be writing if there's a stale dependency.